docs/docs/gatsby-style-guide.md

```diff
@@ -410,11 +415,11 @@ wording.
 
 ### Support software versions
 
-When Gatsby commits to support a specific version of software (e.g. Node 6 and up), this is reflected in documentation. Gatsby documentation should be usable by all people on supported software, which means we don't introduce any commands or practices that can't be used by people on versions we've committed to support. In rare circumstances, we'll consider mentioning a newly introduced command or practice as side notes.
+When Gatsby commits to support a specific version of software (e.g. Node 8 and up), this is reflected in documentation. Gatsby documentation should be usable by all people on supported software, which means we don't introduce any commands or practices that can't be used by people on versions we've committed to support. In rare circumstances, we'll consider mentioning a newly introduced command or practice as side notes.
 
-For example, npm 5.2.0 (which comes with Node 8) introduced a command called `npx` that is not available for versions of Node below 8. Since Gatsby supports Node 6 and up, documentation should only introduce `npx` as an optional command in a note like so:
+For example, npm 5.2.0 (which comes with Node 8) introduced a command called `npx` that is not available for versions of Node below 8. Since Gatsby recommends Node 8 and up but still technically supports Node 6 until the next major release, documentation should point out the requirements for `npx`:
```

Comment:
Wondering how to keep track of what we support throughout the docs. I guess each major change to what we support will require combing the docs to make sure they are up-to-date?

Reply:
I figured it would require a find and replace when a major release happens, and as I come across instances in the docs I'll update them before then. Here's the discussion where Node 6 was "soft dropped": https://github.com/gatsbyjs/rfcs/pull/24